src/screens/crewStationScreen.cpp

```diff
@@ -108,56 +108,98 @@ string CrewStationScreen::populateShortcutsList(CrewPosition position)
 {
     string ret = "";
 
+    bool show_additional_shortcuts_string = false;
+
     // Add shortcuts for this position.
     for (auto binding : sp::io::Keybinding::listAllByCategory(getCrewPositionName(position)))
     {
-        ret += binding->getLabel() + ": " + binding->getHumanReadableKeyName(0) + "\n";
+        if(binding->isBound()){
+            ret += binding->getLabel() + ": " + binding->getHumanReadableKeyName(0) + "\n";
+        }
```

Comment:
please have the code style consistent. All this code is having curly brackets on new lines except for your new code.

Reply:
Not wanting to be annoying, but `else{` isn't fixed ;-) I know it's just minor but it helps me in reading the code faster.